### PR TITLE
added quiz question in index.html file and updated script.js file 

### PR DESCRIPTION
**Pull request by avionicharshit-byte**
added 8th question in the index.html file and also corrected some spelling mistake

### CODE REVIEW

**Comment by Sar-taj107** [comment]
Can you also update the script.js file take reference from existing js file if not please let me know.
And yes, I will approve this PR after review so that it will be counted for your hacktoberfest contribution.

Thanks for the contribution! Do share the repo with your friends and encourage them to contribute for #hactoberfest!

**Comment by avionicharshit-byte** [comment]
yes , I'll update the script.js file too

**Comment by avionicharshit-byte** [comment]
review my pr

File: index.html [hunk 1]
```diff
@@ -37,7 +37,7 @@
             </div>
         </div>
         <div class="question">
-            <p>2. Where is the correwct place to insert a Javascript file?</p>
+            <p>2. Where is the correct place to insert a Javascript file?</p>
```
Comment (Sar-taj107) [inline index.html]: great! thanks for correction

File: script.js [hunk 2]
```diff
@@ -20,5 +26,9 @@ form.addEventListener("submit", event => {
     
     scrollTo(0,0);
     result.classList.remove("hide");
+
+    result.querySelector("p").textContent = `You scored ${score}/8!`;
```
Comment (Sar-taj107) [inline script.js]: just remove this line .
I have update the readme.md file just  once go through the file  you will understand.

File: script.js [hunk 3]
```diff
@@ -7,8 +9,12 @@ form.addEventListener("submit", event => {
     event.preventDefault();
 
     let score=0;
+
+    const userAnswers = [form.q1.value, form.q2.value, form.q3.value, form.q4.value, form.q5.value, form.q6.value, form.q7.value, form.q8.value];
```
Comment (Sar-taj107) [inline script.js]: delete this one also otherwise all are fine

**Review by Sar-taj107 — APPROVED** [review]

**Pull request merged**